policyengine_us/variables/gov/states/ar/tax/income/ar_income_tax_before_non_refundable_credits_indiv.py

```diff
@@ -18,5 +18,11 @@ def formula(person, period, parameters):
         taxable_income = person("ar_taxable_income_indiv", period)
         main_rate = p.rate.calc(taxable_income)
         pre_reduction_tax = main_rate * taxable_income
```

Comment:
What is confusing me is that we are using a `single_amount` structure for the income tax rate
Meaning that we multiply the entire taxable income amount by one rate which increases based on income instead of having a marginal tax rate. 

I can't imagine that this is right